test/powershell/Modules/Microsoft.PowerShell.Management/Get-HotFix.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Get-HotFix Tests" -Tag CI {
+    BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+
+        $skip = $false
+        if (!$IsWindows) {
+            $skip = $true
+        }
+        else {
+            $qfe = Get-CimInstance Win32_QuickFixEngineering
+            if ($qfe.Count -eq 0) {
+                $skip = $true
```

Comment:
I think we should add a comment why we need this.

Reply:
ok